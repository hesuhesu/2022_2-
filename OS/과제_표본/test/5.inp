100
34721 Queenie Angelo
33549 Arletta Grosvenor
45963 Stewart Deforest
24164 Shena Livengood
44929 Chantay Mcniel
00759 Rima Piche
05318 Ashanti Maffei
36012 Thi Weems
03157 Nathan Treanor
12164 Carlo Blanks
01891 Margert Sitler
48134 Ashanti Trinh
46374 Myung Cobbs
49965 Magaret Chenail
39903 Eva Vanzandt
50824 Creola Gebo
37541 Sherrell Kleiner
53770 Ileen Canela
47609 Modesto Pumphrey
08904 Krissy Degen
08375 Tobie Correa
34028 Brittny Mcnichols
38366 Jana Longino
43813 Arron Vance
39571 Jeanette Welles
31293 Mandie Bracero
31860 Tamatha Coombes
08955 Philomena See
35645 Kattie Topp
22685 Sharyn Murley
38947 Christia Latch
54293 Tamika Wolk
17441 Sherlyn Neil
53124 Tarra Goding
22386 Lee Gleaton
11433 Christy Lenoir
13088 Mayra Magnani
23284 Charleen Uhrich
51839 Yvette Gamino
15655 Lyndsey Schoening
42175 Eliseo Sowards
04554 Sybil Kaczynski
40599 Noreen Villa
18640 Darlena Jurgens
09044 Alda Luton
27855 Annelle Nicks
05119 Odette Harsh
25360 Jean Manzi
22096 Sandra Febus
18981 Jane Coop
02430 Miesha Gascon
07443 Mavis Grooms
03660 Cheree Quirion
00585 Rhiannon Wiggs
20016 Shawnda Dearing
06569 Jess Borne
44718 Jann Dell
30674 Katharine Jacques
41361 Rosaura Brisson
50683 Ima Holsapple
05960 Whitney Westhoff
20590 Dorian Tetrault
43132 Tad Casaus
01695 Camilla Wigington
49149 Garret Jimenes
14847 Pamala Gowdy
26110 Sophia Kost
26134 Charlie Orth
09625 Julian Kagawa
46899 Tiffaney Tibbals
30024 Latrina Scheele
21236 Alecia Mathisen
29246 Len Eustice
29595 Kerstin Riegel
17836 Deana Dryden
39397 Rochell Colin
55029 Maryetta Maffucci
10014 Evita Thornburg
12610 Lynsey Haugh
29632 Vernon Spitler
26917 Dudley Donelson
28701 Tiffany Grange
04929 Robbi Enger
24725 Teresa Walcott
48846 Verena Simard
53510 Shaunda Ottley
13904 Cheyenne Weeden
18639 Latosha Scheu
37027 Shanti Syring
23048 Miss Falcon
51411 Angelo Andreas
10463 Gregorio Lavergne
36057 Coretta Sienkiewicz
32658 Millie Willett
35266 Margarita Macaulay
19160 Majorie Kennerly
16632 Mirtha Dahlberg
45348 Ashley Briscoe
52582 Mara Tabor
49796 Verona Akerley
