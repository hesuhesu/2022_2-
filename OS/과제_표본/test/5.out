00585 Rhiannon  Wiggs
00759 Rima      Piche
01695 Camilla   Wigington
01891 Margert   Sitler
02430 Miesha    Gascon
03157 Nathan    Treanor
03660 Cheree    Quirion
04554 Sybil     Kaczynski
04929 Robbi     Enger
05119 Odette    Harsh
05318 Ashanti   Maffei
05960 Whitney   Westhoff
06569 Jess      Borne
07443 Mavis     Grooms
08375 Tobie     Correa
08904 Krissy    Degen
08955 Philomena See
09044 Alda      Luton
09625 Julian    Kagawa
10014 Evita     Thornburg
10463 Gregorio  Lavergne
11433 Christy   Lenoir
12164 Carlo     Blanks
12610 Lynsey    Haugh
13088 Mayra     Magnani
13904 Cheyenne  Weeden
14847 Pamala    Gowdy
15655 Lyndsey   Schoening
16632 Mirtha    Dahlberg
17441 Sherlyn   Neil
17836 Deana     Dryden
18639 Latosha   Scheu
18640 Darlena   Jurgens
18981 Jane      Coop
19160 Majorie   Kennerly
20016 Shawnda   Dearing
20590 Dorian    Tetrault
21236 Alecia    Mathisen
22096 Sandra    Febus
22386 Lee       Gleaton
22685 Sharyn    Murley
23048 Miss      Falcon
23284 Charleen  Uhrich
24164 Shena     Livengood
24725 Teresa    Walcott
25360 Jean      Manzi
26110 Sophia    Kost
26134 Charlie   Orth
26917 Dudley    Donelson
27855 Annelle   Nicks
28701 Tiffany   Grange
29246 Len       Eustice
29595 Kerstin   Riegel
29632 Vernon    Spitler
30024 Latrina   Scheele
30674 Katharine Jacques
31293 Mandie    Bracero
31860 Tamatha   Coombes
32658 Millie    Willett
33549 Arletta   Grosvenor
34028 Brittny   Mcnichols
34721 Queenie   Angelo
35266 Margarita Macaulay
35645 Kattie    Topp
36012 Thi       Weems
36057 Coretta   Sienkiewicz
37027 Shanti    Syring
37541 Sherrell  Kleiner
38366 Jana      Longino
38947 Christia  Latch
39397 Rochell   Colin
39571 Jeanette  Welles
39903 Eva       Vanzandt
40599 Noreen    Villa
41361 Rosaura   Brisson
42175 Eliseo    Sowards
43132 Tad       Casaus
43813 Arron     Vance
44718 Jann      Dell
44929 Chantay   Mcniel
45348 Ashley    Briscoe
45963 Stewart   Deforest
46374 Myung     Cobbs
46899 Tiffaney  Tibbals
47609 Modesto   Pumphrey
48134 Ashanti   Trinh
48846 Verena    Simard
49149 Garret    Jimenes
49796 Verona    Akerley
49965 Magaret   Chenail
50683 Ima       Holsapple
50824 Creola    Gebo
51411 Angelo    Andreas
51839 Yvette    Gamino
52582 Mara      Tabor
53124 Tarra     Goding
53510 Shaunda   Ottley
53770 Ileen     Canela
54293 Tamika    Wolk
55029 Maryetta  Maffucci
